src/main/kotlin/com/numberone/daepiro/domain/auth/filter/JwtFilter.kt

```diff
@@ -0,0 +1,50 @@
+package com.numberone.daepiro.domain.auth.filter
+
+import com.numberone.daepiro.domain.auth.enums.TokenType
+import com.numberone.daepiro.domain.auth.utils.JwtUtils
+import jakarta.servlet.FilterChain
+import jakarta.servlet.http.HttpServletRequest
+import jakarta.servlet.http.HttpServletResponse
+import mu.KotlinLogging
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.http.HttpHeaders.AUTHORIZATION
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken
+import org.springframework.security.core.authority.SimpleGrantedAuthority
+import org.springframework.security.core.context.SecurityContextHolder
+import org.springframework.stereotype.Component
+import org.springframework.web.filter.OncePerRequestFilter
+
+private val logger = KotlinLogging.logger {}
+
+@Component
+class JwtFilter(
+    @Value("\${jwt.secret-key}") private val secretKey: String,
+) : OncePerRequestFilter() {
+    override fun doFilterInternal(
+        request: HttpServletRequest,
+        response: HttpServletResponse,
+        filterChain: FilterChain
+    ) {
+        val jwt = request.getHeader(AUTHORIZATION)
+        if (jwt != null) {
```

Comment:
Bearer 으로 시작하는지도 체크해줘야 할 것 같은데 어떤가요??
```suggestion
        if (jwt != null) {
        if (jwt != null && jwt.startsWith(JwtUtils.PREFIX_BEARER) {
            val token = jwt.substring(JwtUtils.PREFIX_BEARER.length)
        }
```

Reply:
오 그러네요! 감사합니다 👍👍👍